### PR TITLE
Prep 8.0b1 release.

### PR DESCRIPTION
**Pull request by hjoliver**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hjoliver** [comment]
Build tests OK locally :tada:

**Review by kinow — APPROVED** [review]

**Comment by hjoliver** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,8 +18,8 @@ requirements:
     - python
     - 'configurable-http-proxy >=4.1'
     - cylc-flow =={{ version }}
-    - cylc-uiserver ==0.3.0
-    - cylc-ui ==0.3.0
+    - cylc-uiserver ==0.4.0
+    - cylc-ui ==0.4.0
```
Comment (oliver-sanders) [inline recipe/meta.yaml]: Not critical but this dependency is not necessary:

```suggestion
```

**Review by oliver-sanders — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (hjoliver) [reply recipe/meta.yaml]: I know, but it doesn’t really matter, and as we discussed this metapackage is dead-ending soon.

Comment (hjoliver) [reply recipe/meta.yaml]: But OK, I’ll commit the change, it’ll be less confusing for casual observers!

**Pull request merged**